Cargo.lock

```diff
@@ -2292,6 +2304,15 @@ version = "0.11.0+wasi-snapshot-preview1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "9c8d87e72b64a3b4db28d11ce29237c246188f4f51057d65a7eab63b7987e423"
 
+[[package]]
+name = "wasi"
+version = "0.13.3+wasi-0.2.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "26816d2e1a4a36a2940b96c5296ce403917633dff8f3440e9b236ed6f6bacad2"
+dependencies = [
+ "wit-bindgen-rt",
+]
+
 [[package]]
 name = "wasm-bindgen"
```

Comment:
We are not supporting the wasm-based targets. Need to run `cargo tree` and figure out which dependency is pulling in the wasm deps.

Reply:
`cargo tree` reveals that `tempfile` crate relies on a [`getrandom`](https://crates.io/crates/getrandom/0.2.15/dependencies) which declares the `wasi` crate as a dependency. At build time these wasm-related dependencies are not actually compiled though.